tests/TestHelperTrait.php

```diff
@@ -192,9 +192,9 @@ public function getMailjetEmailRepository(): MailjetEmailRepository
         return $this->getRepository(MailjetEmail::class);
     }
 
-    public function getSummaryRepository(): SummaryRepository
```

Comment:
Are you sure that this method is not used anywhere ?
It could be better to create new method and not change existing one.

Reply:
pure mistake!